spec/spec_helper.rb

```diff
@@ -167,3 +167,19 @@ def default_institution
     Agent::AuthorAddress.new(Settings.HARVESTER.INSTITUTION.address.to_hash)
   )
 end
+
+
+WOS_AUTH_WSDL = File.read('spec/fixtures/wos_client/authenticate_wsdl.xml')
+def stub_wos_auth_wsdl
+  raise unless WebOfScience::Client::API_VERSION == '3.0'
```

Comment:
Why this `raise`?  Why would it break at say, `3.1`?

Reply:
I need some way to check or flag that these fixture files need to be updated; it's important that they accurately record what it currently at the URL.  This is the best idea that I came up with so far.

Reply:
Not sure I follow.  If the WSDL changes on the back end, nothing guarantees that `WebOfScience::Client::API_VERSION` changes to match.  I guess if the theory is we use that variable to signal changes, it might be OK, but it also precludes any kind of "minor" SemVer compatibility.

Reply:
Agreed, it's far from perfect.  The only real check is to retrieve it again and do a diff on the fixture file.  If necessary, this PR could add a method to do that once (and only once) when the specs are run.  If we go that way, that change will remove the check on the version number.